### DISCUSSION TITLE
How to translate zod-Messages with Typesafe-i18n in SvelteKit

### DISCUSSION THREAD

**Discussion by Tobfather** [ivanhofer/typesafe-i18n#654] [Q&A] [answered]
Hi, 

I want the error-messages of my zod-schemes to be translated. I have my zod-Schemes in a seperate file zod.ts .

```
import { LL } from '@lang/i18n-svelte';

const lang = get(LL);

export const LoginSchema = z.object({
    email: z.string().min(1, lang.Errors.EmptyField()).email( lang.Errors.InvalidEmail() ),
    password: z.string().min(1, lang.Errors.EmptyField()),
    proceedToUrl: z.string().optional()
});
```

This is working for the default language but when I change the language it doesn't get translated. What would be the correct way of using it?

**Comment by ivanhofer** [comment, marked as answer]
I have added a section to the FAQs. Please take a look an let me know if something is unclear: https://github.com/ivanhofer/typesafe-i18n#how-can-i-have-translated-validation-messages

**Reply by Tobfather** [reply]
Hi Ivan, thank you so much, that works like a charme and makes me happy! :)
